cmd/kogito/command/shared/install_operator.go

```diff
@@ -60,6 +61,11 @@ func SilentlyInstallOperatorIfNotExists(namespace string, operatorImage string,
 func MustInstallOperatorIfNotExists(namespace string, operatorImage string, cli *client.Client, silence bool) (installed bool, err error) {
 	log := context.GetDefaultLogger()
 
+	if util.GetBoolOSEnv("DEBUG") {
```

Comment:
not sure if I followed this, what is the purpose to skip the operator installation if the debug env is set?

Reply:
When we are debugging the CLI using `operator-sdk` feature it's not interesting to have the CLI to install the operator into the namespace since we're running it locally. We might change the env var, though. Like `SKIP_OPERATOR` or `LOCAL`. WDYT?

Reply:
I think SKIP_OPERATOR would be better.